modules/secured-cluster-services-config.adoc

```diff
@@ -303,6 +303,13 @@ If you do not create this account, you must complete future upgrades manually if
 | `monitoring.openshift.enabled`
 | If you set this option to `false`, {product-title} will not set up {osp} monitoring. Defaults to `true` on {osp} 4.
 
+| `network.enableNetworkPolicies`
+a| To provide security at the network level, {product-title-short} creates default `NetworkPolicy` resources in the namespace where secured cluster resources are installed. These network policies allow ingress to specific components on specific ports. If you do not want {product-title-short} to create these policies, set this parameter to `Disabled`. The default value is `Enabled`.
```

Comment:
Sorry for the very late review; the Helm chart value is actually a boolean, with the default being true. You can set it to false to disable writing the network policies. (The Operator part with 'Disabled' and 'Enabled' is correct though).